### PR TITLE
test-renku env list: add gitlab/keycloak credentials

### LINKED ISSUE #398: missing credentials in test init container
currently: 
```
renku-test                                   0/1       Init:Error   0          6m
```

2 fix:

* we need something to make tests easy to run helm from the command line, something like:
```
time helm --set KEYCLOAK_ADMIN_PASSWORD=bla GITLAB_SUDO_TOKEN=blabla helm test renku
```
(the above doesn't work, just to get the idea; i.e. to avoid re-editing `test-renku.yaml`)

### PR DESCRIPTION
**Pull request by fgeorgatos**
Closes #398

### CODE REVIEW

File: charts/renku/templates/secrets.yaml [hunk 1]
```diff
@@ -19,7 +19,8 @@ data:
   gitlab-password: {{ .Values.global.gitlab.postgresPassword | b64enc | quote }}
 {{- end }}
 {{- if .Values.keycloak.enabled }}
-  keycloak-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
+  keycloak-password: {{ .Values.keycloak.keycloak.password | b64enc | quote }}
+  keycloak-postgres-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
   renku-realm.json: {{ include "renku.keycloak.realm" . | indent 4  | b64enc | quote }}
 {{- end }}
   jupyterhub-password: {{ .Values.global.jupyterhub.postgresPassword | b64enc | quote }}
```
Comment (leafty) [inline charts/renku/templates/secrets.yaml]: should we change the other postgress passwords?

Comment (rokroskar) [reply charts/renku/templates/secrets.yaml]: good idea, would make sense imo

Comment (fgeorgatos) [reply charts/renku/templates/secrets.yaml]: @leafty, @rokroskar : ping! what's the action point here?

Comment (rokroskar) [reply charts/renku/templates/secrets.yaml]: for this particular comment -- rename the other password variable names to include "postgres" just as was done with keycloak

**Comment by rokroskar** [comment]
Once the naming is taken care of, we can close this and open a separate issue for fixing the test in the case where the custom renku login theme is used.

File: charts/renku/templates/secrets.yaml [hunk 2]
```diff
@@ -16,10 +16,11 @@ data:
...
+  gitlab-postgres-password: {{ .Values.global.gitlab.postgresPassword | b64enc | quote }}
 {{- end }}
 {{- if .Values.keycloak.enabled }}
-  keycloak-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
+  keycloak-password: {{ .Values.keycloak.keycloak.password | b64enc | quote }}
+  keycloak-postgres-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
   renku-realm.json: {{ include "renku.keycloak.realm" . | indent 4  | b64enc | quote }}
 {{- end }}
-  jupyterhub-password: {{ .Values.global.jupyterhub.postgresPassword | b64enc | quote }}
+  jupyterhub-postgres-password: {{ .Values.global.jupyterhub.postgresPassword | b64enc | quote }}
```
Comment (leafty) [inline charts/renku/templates/secrets.yaml]: Corresponding changes in charts/renku/templates/configmap.yaml and other templates are missing

Comment (fgeorgatos) [reply charts/renku/templates/secrets.yaml]: @leafty : yes, I was suspecting that, this is mid-flight state

**Comment by fgeorgatos** [comment]
@leafty, thanks for last commits.

@rokroskar : let me baptize that: "sdrenku" :) ready for merge?

**Comment by rokroskar** [comment]
yeah LGTM, but don't run `helm test renku` on renkulab because those are actual users.

**Comment by fgeorgatos** [comment]
@rokroskar , @leafty: finally merged, it seems to have passed the testing once master got pulled in.

**Pull request merged**